packages/client/src/internals/fetchHelpers.ts

```diff
@@ -2,11 +2,17 @@ export function getWindow() {
   if (typeof window !== 'undefined') {
     return window;
   }
-  return global;
+  if (typeof global !== 'undefined') {
+    return global;
+  }
+  if (typeof globalThis !== 'undefined') {
```

Comment:
We don't need this function at all if we're using `globalThis`. It's supported in Node v12, Deno, and all the browsers.

Reply:
@KATT, I've reduced the complexity of this code and implemented this suggestion in PR in https://github.com/trpc/trpc/pull/2563